.github/workflows/cypress.yml

```diff
@@ -41,7 +41,9 @@ jobs:
           cache: 'pipenv'
       - name: Install bootstrapper dependencies
         run: pipenv install --dev --deploy
-      - run: pipenv run cookiecutter . --no-input -f
+      - run: |
+          config_file=$(./scripts/vue_or_react.sh)
+          pipenv run cookiecutter . --config-file $config_file --no-input -f
```

Comment:
There is now a function which will detect what type of app is being worked on (details below)